astropy/table/operations.py

```diff
@@ -1275,7 +1275,7 @@ def _join(
 
     out = _get_out_class([left, right])()
 
-    for out_name, dtype, shape in out_descrs:
+    for out_name, _, _ in out_descrs:
```

Comment:
I think it's still useful to name these variable even if we don't use them
```suggestion
    for out_name, _dtype, _shape in out_descrs:
```